src/modules/endpoint/DatabaseConnector.js

```diff
@@ -50,6 +53,16 @@ type ContentEventJsonType = {|
   |}
 |}
 
+type MetaCitiesJsonType = {
+  [string]: {
```

Comment:
Maybe replace `string` with e.g. `CityType` (const type CityType = string) to make it more clear

Reply:
Done. Also replaced the `[string`] for languages.